smtp.go

```diff
@@ -321,6 +324,13 @@ var testHookStartTLS func(*tls.Config) // nil, except for tests
 // functionality. Higher-level packages exist outside of the standard
 // library.
 func SendMail(r *Remote, from string, to []string, msg []byte) error {
+	if r.RateLimiter != nil {
```

Comment:
This is a copy from upstream Golang net/smtp and touching it should be avoided if possible.